src/components/TextLink/__tests__/TextLink.spec.tsx

```diff
@@ -38,23 +38,46 @@ test.describe('TextLink', () => {
   test.describe('non-visual', () => {
     test('href', async ({ mount }) => {
       const component = await mount(
-        <TextLinkForTest
-          href="/test/uri"
-          label="test-label"
-        />,
+        <TextLinkForTest />,
       );
 
       await expect(component).toHaveAttribute('href', '/test/uri');
     });
   });
 
   test.describe('functionality', () => {
+    test('calls native redirect when clicked on', async ({ mount }) => {
+      const testHref = '/test/uri';
+
+      const component = await mount(<TextLinkForTest />);
+      const page = component.page();
+
+      await Promise.all([
+        page.waitForURL(testHref),
+        component.click(),
+      ]);
+
+      expect(page.url()).toContain(testHref);
+    });
+
+    test('calls native redirect when enter pressed', async ({ mount }) => {
+      const testHref = '/test/uri';
```

Comment:
I would move `testHref` higher to share it by all tests. There is one forgotten string not converted to this variable.